HabboHotel/Badges/BadgeManager.cs

```diff
@@ -57,7 +57,7 @@ public async Task GiveBadge(Habbo habbo, string code)
         await connection.ExecuteScalarAsync<int>("REPLACE INTO `user_badges` (`user_id`,`badge_id`,`badge_slot`) VALUES (@userId, @badge, '0');", new

```

Comment:
when badge is given, it is always replaced at slot 0? same at habbo.com? or slot 0 isn't showed in habbo profile?

Reply:
Slot 0 = Not on display (aka just sat in inventory)
If I were to wear the badge it would set it to slot 1, 2, etc

This is correct implementation

Reply:
oh, i see. thank you